netcore/CoreFX.issues.rsp

```diff
@@ -852,6 +843,9 @@
 ##  System.Data.Common.Tests
 ####################################################################
 
+# Invalid IL - IL_00bc: castclass 0x0100000d
+-nomethod System.Data.Tests.SqlTypes.SqlCharsTest.ReadWriteXmlTest
```

Comment:
I think we should create GH issues for each new failing test to keep track of them as same place